server/runtime_nakama_module.go

```diff
@@ -295,6 +296,121 @@ func (n *NakamaModule) usersFetchHandle(l *lua.LState) int {
 	return 1
 }
 
+func (n *NakamaModule) usersUpdate(l *lua.LState) int {
+	updatesTable := l.CheckTable(1)
+	if updatesTable == nil || updatesTable.Len() == 0 {
+		l.ArgError(1, "expects a valid set of user updates")
+		return 0
+	}
+	updatesRaw, ok := convertLuaValue(updatesTable).([]interface{})
+	if !ok {
+		l.ArgError(1, "expects a valid set of user updates")
+		return 0
+	}
+
+	updates := make([]*SelfUpdateOp, 0)
+	for _, updateRaw := range updatesRaw {
+		if updateTable, ok := updateRaw.(map[string]interface{}); !ok {
+			l.ArgError(1, "expects a valid set of user updates")
+			return 0
+		} else {
+			update := &SelfUpdateOp{}
+			// Validate the user ID.
+			if u, ok := updateTable["UserId"]; ok {
```

Comment:
While this is correct, I think there is a nicer, more concise way of writing this code using `Table.ForEach(func(k lua.LValue, v lua.LValue){}`

Also, I think using `lua.LVAsNumber` and `v.String()` is a nicer way than casting data.